source/blog/2015-06-18-ember-data-1-13-released.md

```diff
@@ -0,0 +1,774 @@
+
+While Ember Data has supported JSON API since one of early versions of JSON API through a community [adapter](https://github.com/kurko/ember-json-api) started by 
+@daliwali and maintained by @kurko, now that JSON API has reached 1.0 it is time for Ember Data to uphold it's part of the bargain and make using JSON API a first class experience in Ember Data. 
+
+We have done exactly that, and Ember Data 1.13 adds support throughout the ED stack for JSON Api:
+
+- **Ember Data 1.13 ships with a fully supported JSONApiAdapter and Serializer.** In 2.0, these will become the default Adapter and Serializer
+- **`JSONSerializer` and `RestSerializer` have been refactored and streamlined to return JSON API payloads**
+- **`store.push` now accepts JSON API compliant payload**
+
+Switching to the JSON API format unlocks many new features which will be added in the 2.0 cycle, including better first class pagination, filtering and metadata support.
+
+While **using JSONSerializer, RESTSerializer and ActiveModelSerializer is not deprecated**, we consider JSON API to be the happiest of the happy paths for using Ember Data, and if you are creating a new app today and have control over the API you should be using JSON API, because it is a well designed and comprehensive solution for JSON serialization.
+
+Other major changes in Ember Data 1.13 include:
+
+- [Refactored and simplified Find methods](http://emberjs.com/blog/2015/06/18/ember-data-1-13-released.html#toc_simplified-find-methods)
+- [Adapter level control for identity map caching](http://emberjs.com/blog/2015/06/18/ember-data-1-13-released.html#toc_new-adapter-hooks-for-better-caching)
+- [Refactored and simplified Serializer APIs]()
+- [Switch to using JSONAPI as the internal data storage format](http://emberjs.com/blog/2015/06/18/ember-data-1-13-released.html#toc_new-serializer-api)
+- [Native JSONAPI Serializer](http://emberjs.com/blog/2015/06/18/ember-data-1-13-released.html#toc_json-api-support)
+- [Better Errors handling](http://emberjs.com/blog/2015/06/18/ember-data-1-13-released.html#toc_new-errors-api)
+
+
+
+##Upgrade guide
+
+Ember Data 1.13 is backwards compatible with the beta cycle ED, and there are no
+breaking changes between Ember Data 1.13 and Ember Data beta.19. Ember Data follows Ember.js's lead in not removing any features in this release. The upgrade path should be familiar to Ember.js users, upgrade, remove deprecations, upgrade again.
+
+**You should update your codebase to Ember Data 1.13, remove all the deprecations
+and then move to Ember Data 2.0. It is critically important to do this process stepwise**, as it will give you easy to follow deprecation warnings, and otherwise
```

Comment:
Is "stepwise" the right word here?

Also, break this into two sentences, with the second sentence starting with "Otherwise, your app might fail…"